src/media/stream.ts

```diff
@@ -14,6 +15,8 @@ interface StreamEvents {
  * Base stream class.
  */
 abstract class _Stream {
+  readonly id = uuid();
```

Comment:
Is there a reason not to use [`MediaStream::id`](https://developer.mozilla.org/en-US/docs/Web/API/MediaStream/id)?

Reply:
I thought originally that maybe outputStream would change, but I see now that we had explicitly made it so that the stream itself would always remain the same, so yeah I think I can change the id getter to return `this.outputStream.id`.